Crash opening Android New Overflow Menu

**Bromite version**
Version: 86.0.4240.73 arm64 Android version: 9.0 Device model: SM-G950F (Samsung Galaxy S8)

**Is this bug about the SystemWebView?**
No

**Is the bug reproducible with latest version?**
Yes

**Can the bug be reproduced with corresponding Chromium version?**
No

**Is the bug a crash?**
Yes

**Describe the bug**
After enabling the flags `#tabbed-app-overflow-menu-icons` and `#tabbed-app-overflow-menu-regroup`, the app crashes when trying to open the menu. The solution is uninstall Bromite e then install it again or disable these flags should work too.

**Steps to reproduce the bug:**

1. Open the Chrome Flags (chrome://flags) and enable the flags `#tabbed-app-overflow-menu-icons` and `#tabbed-app-overflow-menu-regroup`
2. Restart the browser
3. Close the browser - close from the background too
4. Open the browser
5. Try to open the menu
6. See error (app crashes)

**Expected behavior**
After enabling those flags, the app should open the menu normally.

**Screenshots**

Flags
![chrome](https://user-images.githubusercontent.com/33721770/95418175-4d3cff00-090d-11eb-9b8f-2bbe1637efad.JPG)

New Overflow Menu
![chrome_new_menu](https://user-images.githubusercontent.com/33721770/95418195-5d54de80-090d-11eb-8fcf-7e80912c0386.JPG)

@csagan5 I don´t know if I did it right, please check if the log below helps you:

```
--------- beginning of crash

F/libc    (29055): Fatal signal 11 (SIGSEGV), code 1, fault addr 0xc in tid 29207 (gads-13)

E/AndroidRuntime( 3541): FATAL EXCEPTION: main

E/AndroidRuntime( 3541): Process: org.bromite.bromite, PID: 3541

E/AndroidRuntime( 3541): java.lang.reflect.UndeclaredThrowableException

E/AndroidRuntime( 3541): 	at $Proxy0.dispatchTouchEvent(Unknown Source)

E/AndroidRuntime( 3541): 	at com.android.internal.policy.impl.PhoneWindow$DecorView.dispatchTouchEvent(PhoneWindow.java:2330)

E/AndroidRuntime( 3541): 	at android.view.View.dispatchPointerEvent(View.java:8671)

E/AndroidRuntime( 3541): 	at android.view.ViewRootImpl$ViewPostImeInputStage.processPointerEvent(ViewRootImpl.java:4193)

E/AndroidRuntime( 3541): 	at android.view.ViewRootImpl$ViewPostImeInputStage.onProcess(ViewRootImpl.java:4059)

E/AndroidRuntime( 3541): 	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:3604)

E/AndroidRuntime( 3541): 	at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:3657)

E/AndroidRuntime( 3541): 	at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:3623)

E/AndroidRuntime( 3541): 	at android.view.ViewRootImpl$AsyncInputStage.forward(ViewRootImpl.java:3740)

E/AndroidRuntime( 3541): 	at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:3631)

E/AndroidRuntime( 3541): 	at android.view.ViewRootImpl$AsyncInputStage.apply(ViewRootImpl.java:3797)

E/AndroidRuntime( 3541): 	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:3604)

E/AndroidRuntime( 3541): 	at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:3657)

E/AndroidRuntime( 3541): 	at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:3623)

E/AndroidRuntime( 3541): 	at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:3631)

E/AndroidRuntime( 3541): 	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:3604)

E/AndroidRuntime( 3541): 	at android.view.ViewRootImpl.deliverInputEvent(ViewRootImpl.java:5912)

E/AndroidRuntime( 3541): 	at android.view.ViewRootImpl.doProcessInputEvents(ViewRootImpl.java:5851)

E/AndroidRuntime( 3541): 	at android.view.ViewRootImpl.enqueueInputEvent(ViewRootImpl.java:5822)

E/AndroidRuntime( 3541): 	at android.view.ViewRootImpl$WindowInputEventReceiver.onInputEvent(ViewRootImpl.java:6002)

E/AndroidRuntime( 3541): 	at android.view.InputEventReceiver.dispatchInputEvent(InputEventReceiver.java:192)

E/AndroidRuntime( 3541): 	at android.os.MessageQueue.nativePollOnce(Native Method)

E/AndroidRuntime( 3541): 	at android.os.MessageQueue.next(MessageQueue.java:143)

E/AndroidRuntime( 3541): 	at android.os.Looper.loop(Looper.java:122)

E/AndroidRuntime( 3541): 	at android.app.ActivityThread.main(ActivityThread.java:5343)

E/AndroidRuntime( 3541): 	at java.lang.reflect.Method.invoke(Native Method)

E/AndroidRuntime( 3541): 	at java.lang.reflect.Method.invoke(Method.java:372)

E/AndroidRuntime( 3541): 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:905)

E/AndroidRuntime( 3541): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:700)

E/AndroidRuntime( 3541): Caused by: java.lang.reflect.InvocationTargetException

E/AndroidRuntime( 3541): 	at java.lang.reflect.Method.invoke(Native Method)

E/AndroidRuntime( 3541): 	at java.lang.reflect.Method.invoke(Method.java:372)

E/AndroidRuntime( 3541): 	at mq.invoke(chromium-ChromePublic.apk-stable-1602278620:7)

E/AndroidRuntime( 3541): 	at java.lang.reflect.Proxy.invoke(Proxy.java:397)

E/AndroidRuntime( 3541): 	... 29 more

E/AndroidRuntime( 3541): Caused by: java.lang.NullPointerException: Attempt to invoke interface method 'android.view.MenuItem android.view.MenuItem.setVisible(boolean)' on a null object reference

E/AndroidRuntime( 3541): 	at cz.s(chromium-ChromePublic.apk-stable-1602278620:3)

E/AndroidRuntime( 3541): 	at cz.e(chromium-ChromePublic.apk-stable-1602278620:82)

E/AndroidRuntime( 3541): 	at kc1.m(chromium-ChromePublic.apk-stable-1602278620:18)

E/AndroidRuntime( 3541): 	at dc1.a(chromium-ChromePublic.apk-stable-1602278620:2)

E/AndroidRuntime( 3541): 	at dc1.onTouch(chromium-ChromePublic.apk-stable-1602278620:9)

E/AndroidRuntime( 3541): 	at android.view.View.dispatchTouchEvent(View.java:8470)

E/AndroidRuntime( 3541): 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2407)

E/AndroidRuntime( 3541): 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2049)

E/AndroidRuntime( 3541): 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2407)

E/AndroidRuntime( 3541): 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2049)

E/AndroidRuntime( 3541): 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2407)

E/AndroidRuntime( 3541): 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2049)

E/AndroidRuntime( 3541): 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2407)

E/AndroidRuntime( 3541): 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2049)

E/AndroidRuntime( 3541): 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2407)

E/AndroidRuntime( 3541): 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2049)

E/AndroidRuntime( 3541): 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2407)

E/AndroidRuntime( 3541): 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2049)

E/AndroidRuntime( 3541): 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2407)

E/AndroidRuntime( 3541): 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2049)

E/AndroidRuntime( 3541): 	at android.view.ViewGroup.dispatchTransformedTouchE

E/AndroidRuntime(18462): FATAL EXCEPTION: main

E/AndroidRuntime(18462): Process: org.bromite.bromite, PID: 18462

E/AndroidRuntime(18462): java.lang.reflect.UndeclaredThrowableException

E/AndroidRuntime(18462): 	at $Proxy0.dispatchTouchEvent(Unknown Source)

E/AndroidRuntime(18462): 	at com.android.internal.policy.impl.PhoneWindow$DecorView.dispatchTouchEvent(PhoneWindow.java:2330)

E/AndroidRuntime(18462): 	at android.view.View.dispatchPointerEvent(View.java:8671)

E/AndroidRuntime(18462): 	at android.view.ViewRootImpl$ViewPostImeInputStage.processPointerEvent(ViewRootImpl.java:4193)

E/AndroidRuntime(18462): 	at android.view.ViewRootImpl$ViewPostImeInputStage.onProcess(ViewRootImpl.java:4059)

E/AndroidRuntime(18462): 	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:3604)

E/AndroidRuntime(18462): 	at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:3657)

E/AndroidRuntime(18462): 	at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:3623)

E/AndroidRuntime(18462): 	at android.view.ViewRootImpl$AsyncInputStage.forward(ViewRootImpl.java:3740)

E/AndroidRuntime(18462): 	at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:3631)

E/AndroidRuntime(18462): 	at android.view.ViewRootImpl$AsyncInputStage.apply(ViewRootImpl.java:3797)

E/AndroidRuntime(18462): 	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:3604)

E/AndroidRuntime(18462): 	at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:3657)

E/AndroidRuntime(18462): 	at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:3623)

E/AndroidRuntime(18462): 	at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:3631)

E/AndroidRuntime(18462): 	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:3604)

E/AndroidRuntime(18462): 	at android.view.ViewRootImpl.deliverInputEvent(ViewRootImpl.java:5912)

E/AndroidRuntime(18462): 	at android.view.ViewRootImpl.doProcessInputEvents(ViewRootImpl.java:5851)

E/AndroidRuntime(18462): 	at android.view.ViewRootImpl.enqueueInputEvent(ViewRootImpl.java:5822)

E/AndroidRuntime(18462): 	at android.view.ViewRootImpl$WindowInputEventReceiver.onInputEvent(ViewRootImpl.java:6002)

E/AndroidRuntime(18462): 	at android.view.InputEventReceiver.dispatchInputEvent(InputEventReceiver.java:192)

E/AndroidRuntime(18462): 	at android.os.MessageQueue.nativePollOnce(Native Method)

E/AndroidRuntime(18462): 	at android.os.MessageQueue.next(MessageQueue.java:143)

E/AndroidRuntime(18462): 	at android.os.Looper.loop(Looper.java:122)

E/AndroidRuntime(18462): 	at android.app.ActivityThread.main(ActivityThread.java:5343)

E/AndroidRuntime(18462): 	at java.lang.reflect.Method.invoke(Native Method)

E/AndroidRuntime(18462): 	at java.lang.reflect.Method.invoke(Method.java:372)

E/AndroidRuntime(18462): 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:905)

E/AndroidRuntime(18462): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:700)

E/AndroidRuntime(18462): Caused by: java.lang.reflect.InvocationTargetException

E/AndroidRuntime(18462): 	at java.lang.reflect.Method.invoke(Native Method)

E/AndroidRuntime(18462): 	at java.lang.reflect.Method.invoke(Method.java:372)

E/AndroidRuntime(18462): 	at mq.invoke(chromium-ChromePublic.apk-stable-1602278620:7)

E/AndroidRuntime(18462): 	at java.lang.reflect.Proxy.invoke(Proxy.java:397)

E/AndroidRuntime(18462): 	... 29 more

E/AndroidRuntime(18462): Caused by: java.lang.NullPointerException: Attempt to invoke interface method 'android.view.MenuItem android.view.MenuItem.setVisible(boolean)' on a null object reference

E/AndroidRuntime(18462): 	at cz.s(chromium-ChromePublic.apk-stable-1602278620:3)

E/AndroidRuntime(18462): 	at cz.e(chromium-ChromePublic.apk-stable-1602278620:82)

E/AndroidRuntime(18462): 	at kc1.m(chromium-ChromePublic.apk-stable-1602278620:18)

E/AndroidRuntime(18462): 	at dc1.a(chromium-ChromePublic.apk-stable-1602278620:2)

E/AndroidRuntime(18462): 	at dc1.onTouch(chromium-ChromePublic.apk-stable-1602278620:9)

E/AndroidRuntime(18462): 	at android.view.View.dispatchTouchEvent(View.java:8470)

E/AndroidRuntime(18462): 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2407)

E/AndroidRuntime(18462): 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2049)

E/AndroidRuntime(18462): 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2407)

E/AndroidRuntime(18462): 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2049)

E/AndroidRuntime(18462): 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2407)

E/AndroidRuntime(18462): 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2049)

E/AndroidRuntime(18462): 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2407)

E/AndroidRuntime(18462): 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2049)

E/AndroidRuntime(18462): 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2407)

E/AndroidRuntime(18462): 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2049)

E/AndroidRuntime(18462): 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2407)

E/AndroidRuntime(18462): 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2049)

E/AndroidRuntime(18462): 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2407)

E/AndroidRuntime(18462): 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2049)

E/AndroidRuntime(18462): 	at android.view.ViewGroup.dispatchTransformedTouch

E/AndroidRuntime(26779): FATAL EXCEPTION: main

E/AndroidRuntime(26779): Process: org.bromite.bromite, PID: 26779

E/AndroidRuntime(26779): java.lang.reflect.UndeclaredThrowableException

E/AndroidRuntime(26779): 	at $Proxy0.dispatchTouchEvent(Unknown Source)

E/AndroidRuntime(26779): 	at com.android.internal.policy.impl.PhoneWindow$DecorView.dispatchTouchEvent(PhoneWindow.java:2330)

E/AndroidRuntime(26779): 	at android.view.View.dispatchPointerEvent(View.java:8671)

E/AndroidRuntime(26779): 	at android.view.ViewRootImpl$ViewPostImeInputStage.processPointerEvent(ViewRootImpl.java:4193)

E/AndroidRuntime(26779): 	at android.view.ViewRootImpl$ViewPostImeInputStage.onProcess(ViewRootImpl.java:4059)

E/AndroidRuntime(26779): 	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:3604)

E/AndroidRuntime(26779): 	at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:3657)

E/AndroidRuntime(26779): 	at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:3623)

E/AndroidRuntime(26779): 	at android.view.ViewRootImpl$AsyncInputStage.forward(ViewRootImpl.java:3740)

E/AndroidRuntime(26779): 	at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:3631)

E/AndroidRuntime(26779): 	at android.view.ViewRootImpl$AsyncInputStage.apply(ViewRootImpl.java:3797)

E/AndroidRuntime(26779): 	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:3604)

E/AndroidRuntime(26779): 	at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:3657)

E/AndroidRuntime(26779): 	at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:3623)

E/AndroidRuntime(26779): 	at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:3631)

E/AndroidRuntime(26779): 	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:3604)

E/AndroidRuntime(26779): 	at android.view.ViewRootImpl.deliverInputEvent(ViewRootImpl.java:5912)

E/AndroidRuntime(26779): 	at android.view.ViewRootImpl.doProcessInputEvents(ViewRootImpl.java:5851)

E/AndroidRuntime(26779): 	at android.view.ViewRootImpl.enqueueInputEvent(ViewRootImpl.java:5822)

E/AndroidRuntime(26779): 	at android.view.ViewRootImpl$WindowInputEventReceiver.onInputEvent(ViewRootImpl.java:6002)

E/AndroidRuntime(26779): 	at android.view.InputEventReceiver.dispatchInputEvent(InputEventReceiver.java:192)

E/AndroidRuntime(26779): 	at android.os.MessageQueue.nativePollOnce(Native Method)

E/AndroidRuntime(26779): 	at android.os.MessageQueue.next(MessageQueue.java:143)

E/AndroidRuntime(26779): 	at android.os.Looper.loop(Looper.java:122)

E/AndroidRuntime(26779): 	at android.app.ActivityThread.main(ActivityThread.java:5343)

E/AndroidRuntime(26779): 	at java.lang.reflect.Method.invoke(Native Method)

E/AndroidRuntime(26779): 	at java.lang.reflect.Method.invoke(Method.java:372)

E/AndroidRuntime(26779): 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:905)

E/AndroidRuntime(26779): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:700)

E/AndroidRuntime(26779): Caused by: java.lang.reflect.InvocationTargetException

E/AndroidRuntime(26779): 	at java.lang.reflect.Method.invoke(Native Method)

E/AndroidRuntime(26779): 	at java.lang.reflect.Method.invoke(Method.java:372)

E/AndroidRuntime(26779): 	at mq.invoke(chromium-ChromePublic.apk-stable-1602278620:7)

E/AndroidRuntime(26779): 	at java.lang.reflect.Proxy.invoke(Proxy.java:397)

E/AndroidRuntime(26779): 	... 29 more

E/AndroidRuntime(26779): Caused by: java.lang.NullPointerException: Attempt to invoke interface method 'android.view.MenuItem android.view.MenuItem.setVisible(boolean)' on a null object reference

E/AndroidRuntime(26779): 	at cz.s(chromium-ChromePublic.apk-stable-1602278620:3)

E/AndroidRuntime(26779): 	at cz.e(chromium-ChromePublic.apk-stable-1602278620:82)

E/AndroidRuntime(26779): 	at kc1.m(chromium-ChromePublic.apk-stable-1602278620:18)

E/AndroidRuntime(26779): 	at dc1.a(chromium-ChromePublic.apk-stable-1602278620:2)

E/AndroidRuntime(26779): 	at dc1.onTouch(chromium-ChromePublic.apk-stable-1602278620:9)

E/AndroidRuntime(26779): 	at android.view.View.dispatchTouchEvent(View.java:8470)

E/AndroidRuntime(26779): 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2407)

E/AndroidRuntime(26779): 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2049)

E/AndroidRuntime(26779): 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2407)

E/AndroidRuntime(26779): 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2049)

E/AndroidRuntime(26779): 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2407)

E/AndroidRuntime(26779): 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2049)

E/AndroidRuntime(26779): 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2407)

E/AndroidRuntime(26779): 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2049)

E/AndroidRuntime(26779): 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2407)

E/AndroidRuntime(26779): 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2049)

E/AndroidRuntime(26779): 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2407)

E/AndroidRuntime(26779): 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2049)

E/AndroidRuntime(26779): 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2407)

E/AndroidRuntime(26779): 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2049)

E/AndroidRuntime(26779): 	at android.view.ViewGroup.dispatchTransformedTouch

E/AndroidRuntime(29075): FATAL EXCEPTION: main

E/AndroidRuntime(29075): Process: org.bromite.bromite, PID: 29075

E/AndroidRuntime(29075): java.lang.reflect.UndeclaredThrowableException

E/AndroidRuntime(29075): 	at $Proxy0.dispatchTouchEvent(Unknown Source)

E/AndroidRuntime(29075): 	at com.android.internal.policy.impl.PhoneWindow$DecorView.dispatchTouchEvent(PhoneWindow.java:2330)

E/AndroidRuntime(29075): 	at android.view.View.dispatchPointerEvent(View.java:8671)

E/AndroidRuntime(29075): 	at android.view.ViewRootImpl$ViewPostImeInputStage.processPointerEvent(ViewRootImpl.java:4193)

E/AndroidRuntime(29075): 	at android.view.ViewRootImpl$ViewPostImeInputStage.onProcess(ViewRootImpl.java:4059)

E/AndroidRuntime(29075): 	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:3604)

E/AndroidRuntime(29075): 	at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:3657)

E/AndroidRuntime(29075): 	at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:3623)

E/AndroidRuntime(29075): 	at android.view.ViewRootImpl$AsyncInputStage.forward(ViewRootImpl.java:3740)

E/AndroidRuntime(29075): 	at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:3631)

E/AndroidRuntime(29075): 	at android.view.ViewRootImpl$AsyncInputStage.apply(ViewRootImpl.java:3797)

E/AndroidRuntime(29075): 	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:3604)

E/AndroidRuntime(29075): 	at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:3657)

E/AndroidRuntime(29075): 	at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:3623)

E/AndroidRuntime(29075): 	at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:3631)

E/AndroidRuntime(29075): 	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:3604)

E/AndroidRuntime(29075): 	at android.view.ViewRootImpl.deliverInputEvent(ViewRootImpl.java:5912)

E/AndroidRuntime(29075): 	at android.view.ViewRootImpl.doProcessInputEvents(ViewRootImpl.java:5851)

E/AndroidRuntime(29075): 	at android.view.ViewRootImpl.enqueueInputEvent(ViewRootImpl.java:5822)

E/AndroidRuntime(29075): 	at android.view.ViewRootImpl$WindowInputEventReceiver.onInputEvent(ViewRootImpl.java:6002)

E/AndroidRuntime(29075): 	at android.view.InputEventReceiver.dispatchInputEvent(InputEventReceiver.java:192)

E/AndroidRuntime(29075): 	at android.os.MessageQueue.nativePollOnce(Native Method)

E/AndroidRuntime(29075): 	at android.os.MessageQueue.next(MessageQueue.java:143)

E/AndroidRuntime(29075): 	at android.os.Looper.loop(Looper.java:122)

E/AndroidRuntime(29075): 	at android.app.ActivityThread.main(ActivityThread.java:5343)

E/AndroidRuntime(29075): 	at java.lang.reflect.Method.invoke(Native Method)

E/AndroidRuntime(29075): 	at java.lang.reflect.Method.invoke(Method.java:372)

E/AndroidRuntime(29075): 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:905)

E/AndroidRuntime(29075): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:700)

E/AndroidRuntime(29075): Caused by: java.lang.reflect.InvocationTargetException

E/AndroidRuntime(29075): 	at java.lang.reflect.Method.invoke(Native Method)

E/AndroidRuntime(29075): 	at java.lang.reflect.Method.invoke(Method.java:372)

E/AndroidRuntime(29075): 	at mq.invoke(chromium-ChromePublic.apk-stable-1602278620:7)

E/AndroidRuntime(29075): 	at java.lang.reflect.Proxy.invoke(Proxy.java:397)

E/AndroidRuntime(29075): 	... 29 more

E/AndroidRuntime(29075): Caused by: java.lang.NullPointerException: Attempt to invoke interface method 'android.view.MenuItem android.view.MenuItem.setVisible(boolean)' on a null object reference

E/AndroidRuntime(29075): 	at cz.s(chromium-ChromePublic.apk-stable-1602278620:3)

E/AndroidRuntime(29075): 	at cz.e(chromium-ChromePublic.apk-stable-1602278620:82)

E/AndroidRuntime(29075): 	at kc1.m(chromium-ChromePublic.apk-stable-1602278620:18)

E/AndroidRuntime(29075): 	at dc1.a(chromium-ChromePublic.apk-stable-1602278620:2)

E/AndroidRuntime(29075): 	at dc1.onTouch(chromium-ChromePublic.apk-stable-1602278620:9)

E/AndroidRuntime(29075): 	at android.view.View.dispatchTouchEvent(View.java:8470)

E/AndroidRuntime(29075): 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2407)

E/AndroidRuntime(29075): 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2049)

E/AndroidRuntime(29075): 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2407)

E/AndroidRuntime(29075): 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2049)

E/AndroidRuntime(29075): 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2407)

E/AndroidRuntime(29075): 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2049)

E/AndroidRuntime(29075): 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2407)

E/AndroidRuntime(29075): 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2049)

E/AndroidRuntime(29075): 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2407)

E/AndroidRuntime(29075): 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2049)

E/AndroidRuntime(29075): 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2407)

E/AndroidRuntime(29075): 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2049)

E/AndroidRuntime(29075): 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2407)

E/AndroidRuntime(29075): 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2049)

E/AndroidRuntime(29075): 	at android.view.ViewGroup.dispatchTransformedTouch

E/AndroidRuntime( 3428): FATAL EXCEPTION: main

E/AndroidRuntime( 3428): Process: org.bromite.bromite, PID: 3428

E/AndroidRuntime( 3428): java.lang.reflect.UndeclaredThrowableException

E/AndroidRuntime( 3428): 	at $Proxy0.dispatchTouchEvent(Unknown Source)

E/AndroidRuntime( 3428): 	at com.android.internal.policy.impl.PhoneWindow$DecorView.dispatchTouchEvent(PhoneWindow.java:2330)

E/AndroidRuntime( 3428): 	at android.view.View.dispatchPointerEvent(View.java:8671)

E/AndroidRuntime( 3428): 	at android.view.ViewRootImpl$ViewPostImeInputStage.processPointerEvent(ViewRootImpl.java:4193)

E/AndroidRuntime( 3428): 	at android.view.ViewRootImpl$ViewPostImeInputStage.onProcess(ViewRootImpl.java:4059)

E/AndroidRuntime( 3428): 	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:3604)

E/AndroidRuntime( 3428): 	at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:3657)

E/AndroidRuntime( 3428): 	at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:3623)

E/AndroidRuntime( 3428): 	at android.view.ViewRootImpl$AsyncInputStage.forward(ViewRootImpl.java:3740)

E/AndroidRuntime( 3428): 	at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:3631)

E/AndroidRuntime( 3428): 	at android.view.ViewRootImpl$AsyncInputStage.apply(ViewRootImpl.java:3797)

E/AndroidRuntime( 3428): 	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:3604)

E/AndroidRuntime( 3428): 	at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:3657)

E/AndroidRuntime( 3428): 	at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:3623)

E/AndroidRuntime( 3428): 	at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:3631)

E/AndroidRuntime( 3428): 	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:3604)

E/AndroidRuntime( 3428): 	at android.view.ViewRootImpl.deliverInputEvent(ViewRootImpl.java:5912)

E/AndroidRuntime( 3428): 	at android.view.ViewRootImpl.doProcessInputEvents(ViewRootImpl.java:5851)

E/AndroidRuntime( 3428): 	at android.view.ViewRootImpl.enqueueInputEvent(ViewRootImpl.java:5822)

E/AndroidRuntime( 3428): 	at android.view.ViewRootImpl$WindowInputEventReceiver.onInputEvent(ViewRootImpl.java:6002)

E/AndroidRuntime( 3428): 	at android.view.InputEventReceiver.dispatchInputEvent(InputEventReceiver.java:192)

E/AndroidRuntime( 3428): 	at android.os.MessageQueue.nativePollOnce(Native Method)

E/AndroidRuntime( 3428): 	at android.os.MessageQueue.next(MessageQueue.java:143)

E/AndroidRuntime( 3428): 	at android.os.Looper.loop(Looper.java:122)

E/AndroidRuntime( 3428): 	at android.app.ActivityThread.main(ActivityThread.java:5343)

E/AndroidRuntime( 3428): 	at java.lang.reflect.Method.invoke(Native Method)

E/AndroidRuntime( 3428): 	at java.lang.reflect.Method.invoke(Method.java:372)

E/AndroidRuntime( 3428): 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:905)

E/AndroidRuntime( 3428): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:700)

E/AndroidRuntime( 3428): Caused by: java.lang.reflect.InvocationTargetException

E/AndroidRuntime( 3428): 	at java.lang.reflect.Method.invoke(Native Method)

E/AndroidRuntime( 3428): 	at java.lang.reflect.Method.invoke(Method.java:372)

E/AndroidRuntime( 3428): 	at mq.invoke(chromium-ChromePublic.apk-stable-1602278620:7)

E/AndroidRuntime( 3428): 	at java.lang.reflect.Proxy.invoke(Proxy.java:397)

E/AndroidRuntime( 3428): 	... 29 more

E/AndroidRuntime( 3428): Caused by: java.lang.NullPointerException: Attempt to invoke interface method 'android.view.MenuItem android.view.MenuItem.setVisible(boolean)' on a null object reference

E/AndroidRuntime( 3428): 	at cz.s(chromium-ChromePublic.apk-stable-1602278620:3)

E/AndroidRuntime( 3428): 	at cz.e(chromium-ChromePublic.apk-stable-1602278620:82)

E/AndroidRuntime( 3428): 	at kc1.m(chromium-ChromePublic.apk-stable-1602278620:18)

E/AndroidRuntime( 3428): 	at dc1.a(chromium-ChromePublic.apk-stable-1602278620:2)

E/AndroidRuntime( 3428): 	at dc1.onTouch(chromium-ChromePublic.apk-stable-1602278620:9)

E/AndroidRuntime( 3428): 	at android.view.View.dispatchTouchEvent(View.java:8470)

E/AndroidRuntime( 3428): 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2407)

E/AndroidRuntime( 3428): 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2049)

E/AndroidRuntime( 3428): 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2407)

E/AndroidRuntime( 3428): 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2049)

E/AndroidRuntime( 3428): 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2407)

E/AndroidRuntime( 3428): 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2049)

E/AndroidRuntime( 3428): 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2407)

E/AndroidRuntime( 3428): 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2049)

E/AndroidRuntime( 3428): 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2407)

E/AndroidRuntime( 3428): 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2049)

E/AndroidRuntime( 3428): 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2407)

E/AndroidRuntime( 3428): 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2049)

E/AndroidRuntime( 3428): 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2407)

E/AndroidRuntime( 3428): 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2049)

E/AndroidRuntime( 3428): 	at android.view.ViewGroup.dispatchTransformedTouchE

E/AndroidRuntime(11768): FATAL EXCEPTION: main

E/AndroidRuntime(11768): Process: org.bromite.bromite, PID: 11768

E/AndroidRuntime(11768): java.lang.reflect.UndeclaredThrowableException

E/AndroidRuntime(11768): 	at $Proxy0.dispatchTouchEvent(Unknown Source)

E/AndroidRuntime(11768): 	at com.android.internal.policy.impl.PhoneWindow$DecorView.dispatchTouchEvent(PhoneWindow.java:2330)

E/AndroidRuntime(11768): 	at android.view.View.dispatchPointerEvent(View.java:8671)

E/AndroidRuntime(11768): 	at android.view.ViewRootImpl$ViewPostImeInputStage.processPointerEvent(ViewRootImpl.java:4193)

E/AndroidRuntime(11768): 	at android.view.ViewRootImpl$ViewPostImeInputStage.onProcess(ViewRootImpl.java:4059)

E/AndroidRuntime(11768): 	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:3604)

E/AndroidRuntime(11768): 	at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:3657)

E/AndroidRuntime(11768): 	at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:3623)

E/AndroidRuntime(11768): 	at android.view.ViewRootImpl$AsyncInputStage.forward(ViewRootImpl.java:3740)

E/AndroidRuntime(11768): 	at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:3631)

E/AndroidRuntime(11768): 	at android.view.ViewRootImpl$AsyncInputStage.apply(ViewRootImpl.java:3797)

E/AndroidRuntime(11768): 	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:3604)

E/AndroidRuntime(11768): 	at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:3657)

E/AndroidRuntime(11768): 	at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:3623)

E/AndroidRuntime(11768): 	at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:3631)

E/AndroidRuntime(11768): 	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:3604)

E/AndroidRuntime(11768): 	at android.view.ViewRootImpl.deliverInputEvent(ViewRootImpl.java:5912)

E/AndroidRuntime(11768): 	at android.view.ViewRootImpl.doProcessInputEvents(ViewRootImpl.java:5851)

E/AndroidRuntime(11768): 	at android.view.ViewRootImpl.enqueueInputEvent(ViewRootImpl.java:5822)

E/AndroidRuntime(11768): 	at android.view.ViewRootImpl$WindowInputEventReceiver.onInputEvent(ViewRootImpl.java:6002)

E/AndroidRuntime(11768): 	at android.view.InputEventReceiver.dispatchInputEvent(InputEventReceiver.java:192)

E/AndroidRuntime(11768): 	at android.os.MessageQueue.nativePollOnce(Native Method)

E/AndroidRuntime(11768): 	at android.os.MessageQueue.next(MessageQueue.java:143)

E/AndroidRuntime(11768): 	at android.os.Looper.loop(Looper.java:122)

E/AndroidRuntime(11768): 	at android.app.ActivityThread.main(ActivityThread.java:5343)

E/AndroidRuntime(11768): 	at java.lang.reflect.Method.invoke(Native Method)

E/AndroidRuntime(11768): 	at java.lang.reflect.Method.invoke(Method.java:372)

E/AndroidRuntime(11768): 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:905)

E/AndroidRuntime(11768): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:700)

E/AndroidRuntime(11768): Caused by: java.lang.reflect.InvocationTargetException

E/AndroidRuntime(11768): 	at java.lang.reflect.Method.invoke(Native Method)

E/AndroidRuntime(11768): 	at java.lang.reflect.Method.invoke(Method.java:372)

E/AndroidRuntime(11768): 	at mq.invoke(chromium-ChromePublic.apk-stable-1602278620:7)

E/AndroidRuntime(11768): 	at java.lang.reflect.Proxy.invoke(Proxy.java:397)

E/AndroidRuntime(11768): 	... 29 more

E/AndroidRuntime(11768): Caused by: java.lang.NullPointerException: Attempt to invoke interface method 'android.view.MenuItem android.view.MenuItem.setVisible(boolean)' on a null object reference

E/AndroidRuntime(11768): 	at cz.s(chromium-ChromePublic.apk-stable-1602278620:3)

E/AndroidRuntime(11768): 	at cz.e(chromium-ChromePublic.apk-stable-1602278620:82)

E/AndroidRuntime(11768): 	at kc1.m(chromium-ChromePublic.apk-stable-1602278620:18)

E/AndroidRuntime(11768): 	at dc1.a(chromium-ChromePublic.apk-stable-1602278620:2)

E/AndroidRuntime(11768): 	at dc1.onTouch(chromium-ChromePublic.apk-stable-1602278620:9)

E/AndroidRuntime(11768): 	at android.view.View.dispatchTouchEvent(View.java:8470)

E/AndroidRuntime(11768): 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2407)

E/AndroidRuntime(11768): 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2049)

E/AndroidRuntime(11768): 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2407)

E/AndroidRuntime(11768): 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2049)

E/AndroidRuntime(11768): 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2407)

E/AndroidRuntime(11768): 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2049)

E/AndroidRuntime(11768): 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2407)

E/AndroidRuntime(11768): 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2049)

E/AndroidRuntime(11768): 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2407)

E/AndroidRuntime(11768): 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2049)

E/AndroidRuntime(11768): 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2407)

E/AndroidRuntime(11768): 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2049)

E/AndroidRuntime(11768): 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2407)

E/AndroidRuntime(11768): 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2049)

E/AndroidRuntime(11768): 	at android.view.ViewGroup.dispatchTransformedTouch

E/AndroidRuntime(32259): FATAL EXCEPTION: main

E/AndroidRuntime(32259): Process: org.bromite.bromite, PID: 32259

E/AndroidRuntime(32259): java.lang.reflect.UndeclaredThrowableException

E/AndroidRuntime(32259): 	at $Proxy0.dispatchTouchEvent(Unknown Source)

E/AndroidRuntime(32259): 	at com.android.internal.policy.impl.PhoneWindow$DecorView.dispatchTouchEvent(PhoneWindow.java:2330)

E/AndroidRuntime(32259): 	at android.view.View.dispatchPointerEvent(View.java:8671)

E/AndroidRuntime(32259): 	at android.view.ViewRootImpl$ViewPostImeInputStage.processPointerEvent(ViewRootImpl.java:4193)

E/AndroidRuntime(32259): 	at android.view.ViewRootImpl$ViewPostImeInputStage.onProcess(ViewRootImpl.java:4059)

E/AndroidRuntime(32259): 	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:3604)

E/AndroidRuntime(32259): 	at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:3657)

E/AndroidRuntime(32259): 	at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:3623)

E/AndroidRuntime(32259): 	at android.view.ViewRootImpl$AsyncInputStage.forward(ViewRootImpl.java:3740)

E/AndroidRuntime(32259): 	at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:3631)

E/AndroidRuntime(32259): 	at android.view.ViewRootImpl$AsyncInputStage.apply(ViewRootImpl.java:3797)

E/AndroidRuntime(32259): 	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:3604)

E/AndroidRuntime(32259): 	at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:3657)

E/AndroidRuntime(32259): 	at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:3623)

E/AndroidRuntime(32259): 	at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:3631)

E/AndroidRuntime(32259): 	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:3604)

E/AndroidRuntime(32259): 	at android.view.ViewRootImpl.deliverInputEvent(ViewRootImpl.java:5912)

E/AndroidRuntime(32259): 	at android.view.ViewRootImpl.doProcessInputEvents(ViewRootImpl.java:5851)

E/AndroidRuntime(32259): 	at android.view.ViewRootImpl.enqueueInputEvent(ViewRootImpl.java:5822)

E/AndroidRuntime(32259): 	at android.view.ViewRootImpl$WindowInputEventReceiver.onInputEvent(ViewRootImpl.java:6002)

E/AndroidRuntime(32259): 	at android.view.InputEventReceiver.dispatchInputEvent(InputEventReceiver.java:192)

E/AndroidRuntime(32259): 	at android.os.MessageQueue.nativePollOnce(Native Method)

E/AndroidRuntime(32259): 	at android.os.MessageQueue.next(MessageQueue.java:143)

E/AndroidRuntime(32259): 	at android.os.Looper.loop(Looper.java:122)

E/AndroidRuntime(32259): 	at android.app.ActivityThread.main(ActivityThread.java:5343)

E/AndroidRuntime(32259): 	at java.lang.reflect.Method.invoke(Native Method)

E/AndroidRuntime(32259): 	at java.lang.reflect.Method.invoke(Method.java:372)

E/AndroidRuntime(32259): 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:905)

E/AndroidRuntime(32259): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:700)

E/AndroidRuntime(32259): Caused by: java.lang.reflect.InvocationTargetException

E/AndroidRuntime(32259): 	at java.lang.reflect.Method.invoke(Native Method)

E/AndroidRuntime(32259): 	at java.lang.reflect.Method.invoke(Method.java:372)

E/AndroidRuntime(32259): 	at mq.invoke(chromium-ChromePublic.apk-stable-1602278620:7)

E/AndroidRuntime(32259): 	at java.lang.reflect.Proxy.invoke(Proxy.java:397)

E/AndroidRuntime(32259): 	... 29 more

E/AndroidRuntime(32259): Caused by: java.lang.NullPointerException: Attempt to invoke interface method 'android.view.MenuItem android.view.MenuItem.setVisible(boolean)' on a null object reference

E/AndroidRuntime(32259): 	at cz.s(chromium-ChromePublic.apk-stable-1602278620:3)

E/AndroidRuntime(32259): 	at cz.e(chromium-ChromePublic.apk-stable-1602278620:82)

E/AndroidRuntime(32259): 	at kc1.m(chromium-ChromePublic.apk-stable-1602278620:18)

E/AndroidRuntime(32259): 	at dc1.a(chromium-ChromePublic.apk-stable-1602278620:2)

E/AndroidRuntime(32259): 	at dc1.onTouch(chromium-ChromePublic.apk-stable-1602278620:9)

E/AndroidRuntime(32259): 	at android.view.View.dispatchTouchEvent(View.java:8470)

E/AndroidRuntime(32259): 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2407)

E/AndroidRuntime(32259): 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2049)

E/AndroidRuntime(32259): 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2407)

E/AndroidRuntime(32259): 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2049)

E/AndroidRuntime(32259): 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2407)

E/AndroidRuntime(32259): 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2049)

E/AndroidRuntime(32259): 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2407)

E/AndroidRuntime(32259): 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2049)

E/AndroidRuntime(32259): 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2407)

E/AndroidRuntime(32259): 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2049)

E/AndroidRuntime(32259): 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2407)

E/AndroidRuntime(32259): 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2049)

E/AndroidRuntime(32259): 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2407)

E/AndroidRuntime(32259): 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2049)

E/AndroidRuntime(32259): 	at android.view.ViewGroup.dispatchTransformedTouch
```

unfortunately I don't have any crashes. 
![image](https://user-images.githubusercontent.com/29201891/95676564-302f5700-0bbf-11eb-8c38-f9e9bb17d531.png)


We can do as last time by decompiling the apk or provide a debug apk version to @frankjrp
This I imagine the stack to look for
```
E/AndroidRuntime(32259): Caused by: java.lang.NullPointerException: Attempt to invoke interface method 'android.view.MenuItem android.view.MenuItem.setVisible(boolean)' on a null object reference
E/AndroidRuntime(32259): 	at cz.s(chromium-ChromePublic.apk-stable-1602278620:3)
E/AndroidRuntime(32259): 	at cz.e(chromium-ChromePublic.apk-stable-1602278620:82)
E/AndroidRuntime(32259): 	at kc1.m(chromium-ChromePublic.apk-stable-1602278620:18)
E/AndroidRuntime(32259): 	at dc1.a(chromium-ChromePublic.apk-stable-1602278620:2)
E/AndroidRuntime(32259): 	at dc1.onTouch(chromium-ChromePublic.apk-stable-1602278620:9)
E/AndroidRuntime(32259): 	at android.view.View.dispatchTouchEvent(View.java:8470)
```

![image](https://user-images.githubusercontent.com/29201891/95676944-5d313900-0bc2-11eb-882a-237931dbec1f.png)
![image](https://user-images.githubusercontent.com/29201891/95676951-6c17eb80-0bc2-11eb-8f0a-1f1f30958af8.png)
![image](https://user-images.githubusercontent.com/29201891/95676954-7afe9e00-0bc2-11eb-8b74-f142f7d0817b.png)
![image](https://user-images.githubusercontent.com/29201891/95676963-8eaa0480-0bc2-11eb-8ded-ad2c9c6a518e.png)

--- a/build/patches/Add-exit-menu-item.patch
+++ b/build/patches/Add-exit-menu-item.patch
@@ -5,10 +5,11 @@ Subject: Add exit menu item
 Corrected Exit functionality
 ---
  chrome/android/java/res/menu/main_menu.xml                  | 4 ++++
+ chrome/android/java/res/menu/main_menu_regroup.xml          | 3 +++
  .../src/org/chromium/chrome/browser/app/ChromeActivity.java | 6 ++++++
  .../chrome/browser/init/ChromeLifetimeController.java       | 6 +++++-
  .../browser/ui/android/strings/android_chrome_strings.grd   | 3 +++
- 4 files changed, 18 insertions(+), 1 deletion(-)
+ 5 files changed, 21 insertions(+), 1 deletion(-)
 
 diff --git a/chrome/android/java/res/menu/main_menu.xml b/chrome/android/java/res/menu/main_menu.xml
 --- a/chrome/android/java/res/menu/main_menu.xml
@@ -31,6 +32,19 @@ diff --git a/chrome/android/java/res/menu/main_menu.xml b/chrome/android/java/re
      </group>
  
      <!-- Items shown only in the tab switcher when start surface is enabled -->
+diff --git a/chrome/android/java/res/menu/main_menu_regroup.xml b/chrome/android/java/res/menu/main_menu_regroup.xml
+--- a/chrome/android/java/res/menu/main_menu_regroup.xml
++++ b/chrome/android/java/res/menu/main_menu_regroup.xml
+@@ -133,6 +133,9 @@
+             android:icon="@drawable/vr_headset" />
+         <item android:id="@+id/managed_by_menu_id"
+             android:title="@string/managed" />
++        <item android:id="@+id/exit_id"
++            android:title="@string/menu_exit" 
++            android:icon="@drawable/ic_cloud_offline_24dp"/>
+     </group>
+ 
+     <!-- Items shown only in the tab switcher -->
 diff --git a/chrome/android/java/src/org/chromium/chrome/browser/app/ChromeActivity.java b/chrome/android/java/src/org/chromium/chrome/browser/app/ChromeActivity.java
 --- a/chrome/android/java/src/org/chromium/chrome/browser/app/ChromeActivity.java
 +++ b/chrome/android/java/src/org/chromium/chrome/browser/app/ChromeActivity.java
@@ -83,6 +97,3 @@ diff --git a/chrome/browser/ui/android/strings/android_chrome_strings.grd b/chro
        <message name="IDS_MENU_CLOSE_ALL_TABS" desc="Menu item for closing all open tabs. [CHAR-LIMIT=27]">
          Close all tabs
        </message>
--- 
-2.17.1
-

--- a/build/patches/Add-exit-menu-item.patch
+++ b/build/patches/Add-exit-menu-item.patch
@@ -41,7 +41,7 @@ diff --git a/chrome/android/java/res/menu/main_menu_regroup.xml b/chrome/android
              android:title="@string/managed" />
 +        <item android:id="@+id/exit_id"
 +            android:title="@string/menu_exit" 
-+            android:icon="@drawable/ic_cloud_offline_24dp"/>
++            android:icon="@drawable/ic_exit_to_app_white_24dp"/>
      </group>
  
      <!-- Items shown only in the tab switcher -->

--- a/build/patches/Add-menu-item-to-view-source.patch
+++ b/build/patches/Add-menu-item-to-view-source.patch
@@ -5,12 +5,13 @@ Subject: Add menu item to view source
 ---
  chrome/android/java/res/menu/custom_tabs_menu.xml  |  3 +++
  chrome/android/java/res/menu/main_menu.xml         |  2 ++
+ chrome/android/java/res/menu/main_menu_regroup.xml |  3 +++
  .../chrome/browser/ChromeTabbedActivity.java       |  2 ++
  .../chrome/browser/app/ChromeActivity.java         |  2 ++
  .../app/appmenu/AppMenuPropertiesDelegateImpl.java | 14 ++++++++++++++
  .../CustomTabAppMenuPropertiesDelegate.java        |  1 +
  .../ui/android/strings/android_chrome_strings.grd  |  3 +++
- 7 files changed, 27 insertions(+)
+ 8 files changed, 30 insertions(+)
 
 diff --git a/chrome/android/java/res/menu/custom_tabs_menu.xml b/chrome/android/java/res/menu/custom_tabs_menu.xml
 --- a/chrome/android/java/res/menu/custom_tabs_menu.xml
@@ -34,6 +35,19 @@ diff --git a/chrome/android/java/res/menu/main_menu.xml b/chrome/android/java/re
              android:icon="@drawable/ic_add_to_home_screen" />
 +        <item android:id="@+id/view_source_id"
 +            android:title="@string/view_source" />
+         <item android:id="@+id/open_webapk_id"
+             android:title="@string/menu_open_webapk"
+             android:icon="@drawable/ic_add_to_home_screen" />
+diff --git a/chrome/android/java/res/menu/main_menu_regroup.xml b/chrome/android/java/res/menu/main_menu_regroup.xml
+--- a/chrome/android/java/res/menu/main_menu_regroup.xml
++++ b/chrome/android/java/res/menu/main_menu_regroup.xml
+@@ -97,6 +97,9 @@
+         <item android:id="@+id/add_to_homescreen_id"
+             android:title="@string/menu_add_to_homescreen"
+             android:icon="@drawable/ic_add_to_home_screen" />
++        <item android:id="@+id/view_source_id"
++            android:title="@string/view_source"
++            android:icon="@drawable/ic_drive_document_24dp" />
          <item android:id="@+id/open_webapk_id"
              android:title="@string/menu_open_webapk"
              android:icon="@drawable/ic_add_to_home_screen" />
@@ -116,6 +130,3 @@ diff --git a/chrome/browser/ui/android/strings/android_chrome_strings.grd b/chro
        <message name="IDS_PREFS_GOOGLE_SERVICES" desc="Title for the group of preferences that control non-personalized Google services. This group contains preferences for data that is not tied to user's Google Account. [CHAR-LIMIT=40]">
          Google services
        </message>
--- 
-2.17.1
-

